Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-multiple-answers/questions-multiple-answers.component.ts

```diff
@@ -1,9 +1,96 @@
 ﻿import { Component, OnInit, ViewChild } from '@angular/core';
-
+import { MdSnackBar } from '@angular/material';
+import { CategoryService } from '../categories.service';
+import { Category } from '../category.model';
+import { QuestionBase } from '../question';
+import { Router } from '@angular/router';
+import { QuestionsService } from '../questions.service';
+import { DifficultyLevel } from '../enum-difficultylevel';
+import { SingleMultipleAnswerQuestionOption } from '../single-multiple-answer-question-option.model';
+import { Observable } from 'rxjs';
 @Component({
     moduleId: module.id,
     selector: 'questions-multiple-answers',
     templateUrl: 'questions-multiple-answers.html'
 })
 
-export class QuestionsMultipleAnswersComponent { }
+export class QuestionsMultipleAnswersComponent {
```

Comment:
I can see everything twice. The Just name is different. Can't, you handle in one file?